server/src/middleware/file-upload.interceptor.ts

```diff
@@ -32,7 +32,7 @@ export function getFiles(files: UploadFiles) {
 }
 
 export enum Route {
-  ASSET = 'asset',
+  ASSET = 'assets',
```

Comment:
I think you should also move canUploadFIle getUploadFilename getUploadFolder from asset service to asset-media service.